### DISCUSSION TITLE
Support alpha channel for HEX colors

### DISCUSSION THREAD

**Discussion by LukasWieditz** [spatie/color#51] [Ideas]
With [all modern browsers](https://caniuse.com/css-rrggbbaa) having added support for 8 digit hex notations, an update to the library to support alpha channels in the HEX color system would be nice.

**Comment by acalvino4** [comment]
related to https://github.com/spatie/color/discussions/59

**Comment by LukasWieditz** [comment]
This is implemented with #54
